package-lock.json

```diff
@@ -1,15315 +1,8 @@
 {
   "name": "pomagamukrainie",
   "version": "0.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
This is not correct, we need to remain on version 2 of the lockfile. Please use a recent Node.js version which will include a more recent npm version by default, alleviating this issue. This is important for all contributions to this repository that affect npm dependencies.

Reply:
Ideally, revert all the changes in this file, then update Node.js, then run `npm install again` so that it regenerates `package-lock.json` based on what's on `main`, not on this version of the lockfile.

I can also do it myself if you prefer; let me know, @luziczek.

Reply:
I will do that in a moment.